exporter/zipkinexporter/zipkin_test.go

```diff
@@ -155,14 +157,18 @@ func TestZipkinExporter_roundtripJSON(t *testing.T) {
 	}))
 	defer cst.Close()
 
-	tes, err := newZipkinExporter(cst.URL, "", time.Millisecond, "json")
+	config := &Config{
+		URL:    cst.URL,
+		Format: "json",
+	}
+	tes, err := createZipkinExporter(zap.NewNop(), config)
 	assert.NoError(t, err)
 	require.NotNil(t, tes)
 
 	// The test requires the spans from zipkinSpansJSONJavaLibrary to be sent in a single batch, use
 	// a mock to ensure that this happens as intended.
 	mzr := newMockZipkinReporter(cst.URL)
-	tes.reporter = mzr
+	// tes.reporter = mzr
```

Comment:
The commented line seems fail the test. @pjanotti , could you provide any guidance on this? Thanks.

Reply:
Oh the mock for the test needs a reporter, likely you can instantiate the actual zipkin report here and use it with the mock used in the test. I will check that for you in couple of hours.